### PR TITLE
chore: Adds the ability to see the existing filter when we apply a new filter

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/product/issues/782

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
https://www.loom.com/share/125094c5cb7842b6840daa286db4059f

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/ChatList.vue [hunk 1]
```diff
@@ -773,6 +791,74 @@ export default {
     onContextMenuToggle(state) {
       this.isContextMenuOpen = state;
     },
+    initializeExistingFilterToModal() {
```
Comment (nithindavid) [inline app/javascript/dashboard/components/ChatList.vue:794]: Can we move this function to `app/javascript/shared/mixins/filterMixin.js`

**Review by nithindavid — COMMENTED** [review]
Tested and working for all cases. One minor comment added.

File: app/javascript/dashboard/components/ChatList.vue (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/components/ChatList.vue]: Done https://github.com/chatwoot/chatwoot/pull/6310/commits/d8dfe7e5b92e12e9810a4f00a9402602e21baacb

**Comment by nithindavid** [comment]
@iamsivin Please address the codeclimate issue.

**Comment by iamsivin** [comment]
> @iamsivin Please address the codeclimate issue.

@nithindavid Done https://github.com/chatwoot/chatwoot/pull/6310/commits/4ebe42e6d0dd5d5e80e41cd9ec4d7fd0779bd1d6

**Review by nithindavid — APPROVED** [review]

**Review by nithindavid — APPROVED** [review]

**Pull request merged**